cypress/integration/options.cy.js

```diff
@@ -0,0 +1,110 @@
+import {
+    selectProgramDimensions,
+    INPUT_EVENT,
+    selectPeriod,
+    FIXED,
+    getPreviousYearStr,
+    getLineListTable,
+} from '../helpers/index.js'
+import { EXTENDED_TIMEOUT } from '../support/util.js'
+
+describe('options', () => {
+    it('sets comfortable display density', () => {
+        cy.visit('#/R4wAb2yMLik', EXTENDED_TIMEOUT)
```

Comment:
Just a note, in addition to making sure that old AOs work we should also test the complete flow for setting these options from scratch. Might be good to add a `// TODO: ` comment as a reminder.

Reply:
My bad, I thought the opened AO contained set options already, but I see now that they are just plain AOs (since any AO can use these options anyway), so disregard my comment above.